google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,238 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.iam.admin.spi.v1
+  python:
+    package_name: google.cloud.gapic.iam.admin.v1
```

Comment:
Please change to `google.cloud.gapic.iam-admin.v1`. Unfortunately, this affects https://github.com/googleapis/api-client-staging/pull/20.